template/common-parameter-meta.txt

```diff
@@ -0,0 +1,37 @@
+        memory_gb: "RAM to allocate for task"
+        disk_size_gb: "Disk space to allocate for task"
+        modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs."
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs."
+        ncpu: "Number of cores to allocate for task"
+        detect_nproc: "Use all available cores? Recommended for cloud environments. Not recommended for cluster environments."
+        max_retries: "<for a task>Number of times to retry in case of failure"
+        max_retries: "<for a workflow>Number of times to retry failed steps. Overrides task level defaults."
```

Comment:
```suggestion
        max_retries: "<for a task> Number of times to retry in case of failure"
        max_retries: "<for a workflow> Number of times to retry failed steps. Overrides task level defaults."
```